ops/terraform/env/prod/stateful/main.tf

```diff
@@ -16,6 +16,13 @@ module "stateful" {
     allocated_storage = 16000
   }
 
+  db_params = [
+    {name="max_worker_processes", value="16", apply_on_reboot=true},
+    {name="random_page_cost", value="1", apply_on_reboot=false},
+    {name="temp_buffers", value="8192", apply_on_reboot=false},
+    {name="work_mem", value="32768", apply_on_reboot=false}
```

Comment:
FYI: We currently use 40 DB connections per app server (overkill, but whatev) * max-ASG-size of 24 * 32MB = 30.7 GB RAM used for `work_mem`.

Which is fine! Just wanted to note it.

Reply:
Wont this only be per AZ, so 8/24 instances in that AZ will connect to their respective replica.

Reply:
Good call!